2-mandatory/6-budgets.js

```diff
@@ -2,17 +2,36 @@
 Get Sum of People's Budget
 Create the function that takes an array with objects and returns the sum of people's budgets.
 For example:
-getBudgets([
-  { name: "John",  age: 21, budget: 29000 },
-  { name: "Steve",  age: 32, budget: 32000 },
-  { name: "Martin",  age: 16, budget: 1600 }
-])
+
 Should give return the answer of 62600.
 **/
+let bdg = [{
+    name: "John",
+    age: 21,
+    budget: 29000
+  },
+  {
+    name: "Steve",
+    age: 32,
+    budget: 32000
+  },
+  {
+    name: "Martin",
+    age: 16,
+    budget: 1600
+  }
+];
 
 function getBudgets(peopleArray) {
+  let totalBudget = 0;
+  let result = peopleArray.forEach(obj => totalBudget = totalBudget + obj.budget);
```

Comment:
To neaten this up a bit: the forEach is good, it works well, but what is the "result"? Where are we using it?